common/web/keyboard-processor/src/text/kbdInterface.ts

```diff
@@ -529,6 +529,43 @@ namespace com.keyman.text {
       }
     }
 
+    /**
+     * Maps a KeyEvent's modifiers to their appropriate value for key-rule evaluation
+     * based on the rule's specified target modifier set.
+     *
+     * Mostly used to correct chiral OSK-keys targeting non-chiral rules.
+     * @param e The source KeyEvent
+     * @returns
+     */
+    private static getEventRuleModifiers(e: KeyEvent, targetModifierMask: number): number {
+      const CHIRAL_ALT  = Codes.modifierCodes["LALT"]  | Codes.modifierCodes["RALT"];
+      const CHIRAL_CTRL = Codes.modifierCodes["LCTRL"] | Codes.modifierCodes["RCTRL"];
+
+      let modifiers = e.Lmodifiers;
+
+      // If the target rule does not use chiral alt...
+      if(!(targetModifierMask & CHIRAL_ALT)) {
+        const alt_intersection  = modifiers & CHIRAL_ALT;
+
+        if(alt_intersection) {
+          // Undo the chiral part          and replace with non-chiral.
+          modifiers ^= alt_intersection  | Codes.modifierCodes["ALT"];
+        }
```

Comment:
```suggestion
        const altIntersection  = modifiers & CHIRAL_ALT;

        if(altIntersection) {
          // Undo the chiral part         and replace with non-chiral.
          modifiers ^= altIntersection  | Codes.modifierCodes["ALT"];
        }
```

Should be using `camelCase` :grin: